bot/bot.py

```diff
@@ -107,7 +107,7 @@ def register_conversation_commands(dispatcher):
             ),
             CallbackQueryHandler(
                 handlers.status.add_status_user.start,
-                pattern="^add_status_user$"
```

Comment:
This change isn't great, as well as some similar naming further below, but it's OK for now...